### PR TITLE
add velo kuji-eth

### PR DESCRIPTION
**Pull request by BustinJieber420**

### CODE REVIEW

File: src/data/optimism/velodromeLpPools.json [hunk 1]
```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "name": "velodrome-v2-kuji-weth",
+    "address": "0x1e7c5572B3D5942Cf5dabf61f5Ba3FbC6B65e921",
```
Comment (roman-monk) [inline src/data/optimism/velodromeLpPools.json:4]: definitely not kuji lp

Comment (BustinJieber420) [reply src/data/optimism/velodromeLpPools.json]: f. Thought I tripled checked everything. Fixed

File: packages/address-book/address-book/optimism/tokens/tokens.ts [hunk 2]
```diff
@@ -1299,6 +1299,20 @@ const _tokens = {
...
+    oracleId: 'KUJI',
+    address: '0x3A18dcC9745eDcD1Ef33ecB93b0b6eBA5671e7Ca',
+    chainId: 10,
+    decimals: 6,
+    logoURI: '',
+    website: 'https://kujira.network/',
+    description:
+      'Kujira is a decentralized ecosystem revolutionizing payment infrastructure and providing sustainable FinTech solutions to protocols, builders, and web 3 users. An innovation hub of revenue-generating products with great user experience that allow retail investors to access opportunities previously only available to the wealthy and elite.',
+    documentation: 'https://docs.kujira.app/',
+    bridge: 'native',
```
Comment (MirthFutures) [inline packages/address-book/address-book/optimism/tokens/tokens.ts:1314]: Bridge is Axelar

Comment (BustinJieber420) [reply packages/address-book/address-book/optimism/tokens/tokens.ts]: "Kujira native asset" fooled me. One has to verify everything in this space

**Pull request merged**